src/main/java/org/folio/rest/impl/LoansAPI.java

```diff
@@ -293,9 +294,30 @@ private Errors moreThanOneOpenLoanError(Loan entity) {
   }
 
   private boolean isMultipleOpenLoanError(AsyncResult<Response> reply) {
-    return reply.succeeded()
-      && reply.result().getStatus() == 400
-      && reply.result().getEntity().toString().contains("loan_itemid_idx_unique");
+    String message = "";
+
+    if (reply.succeeded() && reply.result().getStatus() >= 400 &&
+      reply.result().getStatus() < 600 && reply.result().hasEntity()) {
+
+      // When entity is an instance of Errors, the getEntity().toString()
+      // will return an object identifier. Process the object to correctly
+      // parse the message.
+      if (reply.result().getEntity() instanceof Errors) {
+        Errors errors = (Errors) reply.result().getEntity();
+
+        for (int i = 0; i < errors.getErrors().size(); i++) {
+          Error error = errors.getErrors().get(i);
+
+          if (error.getType() == RTFConsts.VALIDATION_FIELD_ERROR)
```

Comment:
Maybe not rely on a specific error type. I don't think that we will have multiply errors here.

Reply:
I was wondering the same and was uncertain so I left it to review. This will be done.